packages/api/src/clickhouse/searchQueryParser.ts

```diff
@@ -652,3 +652,21 @@ export const buildSearchQueryWhereCondition = async ({
   builder.teamId = teamId;
   return await builder.timestampInBetween(startTime, endTime).build();
 };
+
+export const buildPostGroupWhereCondition = ({ query }: { query: string }) => {
```

Comment:
nit: I wonder why we don't pass query string directly

Reply:
Oh like why use a kwargs-like setup? I suspect this function will grow over time and copied the `buildSearchQueryWhereCondition` format.